### PR TITLE
fix model creator in pythorch bigdl

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 1]
```diff
@@ -263,18 +263,19 @@ def __init__(self, model, loss, optimizer, config=None, metrics=None, model_dir=
...
         if self.loss is None:
             self.loss = TorchLoss()
         else:
             self.loss = TorchLoss.from_pytorch(loss)
-        if isinstance(self.model, types.FunctionType):
-            self.model = self.model(self.config)
+        if isinstance(model, types.FunctionType):
+            model = model(self.config)
+            def model_creator():
+                return model(self.config)
```
Comment (qiuxin2012) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:276]:
```
def model_creator():                
    return model(self.config)
model = model_creator

```

File: pyzoo/zoo/orca/learn/pytorch/estimator.py [hunk 2]
```diff
@@ -289,7 +290,10 @@ def __init__(self, model, loss, optimizer, config=None, metrics=None, model_dir=
         self.log_dir = None
         self.app_name = None
         self.model_dir = model_dir
-        self.model = TorchModel.from_pytorch(self.model)
+        if isinstance(model, types.FunctionType):
+            self.model = TorchModel.from_pytorch(model_creator)
+        else:
+            self.model = TorchModel.from_pytorch(model)
```
Comment (qiuxin2012) [inline pyzoo/zoo/orca/learn/pytorch/estimator.py:296]: just `self.model = TorchModel.from_pytorch(model)`

File: pyzoo/zoo/orca/learn/pytorch/estimator.py (hunk 1 above)
Comment (Le-Zheng) [reply pyzoo/zoo/orca/learn/pytorch/estimator.py:275]:
```
    def model_creator():
>       return model(self.config)
E       TypeError: model_creator() takes 0 positional arguments but 1 was given
```

Comment (qiuxin2012) [reply pyzoo/zoo/orca/learn/pytorch/estimator.py:275]: try `return model(config)`

**Comment by Le-Zheng** [comment]
Method defined in class must pass 'self' or use `@staticmethod`. Use 'self' in this pr. 

jenkins passed: http://10.239.176.111:18888/job/ZOO-PR-Validation/5814/

**Pull request merged**